### ISSUE TITLE
Indicators on track graph: What to show when results are reported but first target period isn't over

### ISSUE THREAD

**Issue by jennyhavah** [mercycorps/TolaActivity#910]
Discovered by Jon Prettyman. In a scenario where results are reported but the first target period is not yet over, we currently show a sadface and message implying that there are no results.

Instead, we want to show an empty gauge with the message, "Unavailable until the first target period ends"

@axoplasm The only text change I'd suggest is to change "your" to "the" since people outside the immediate program team will also be viewing this -- a country director or regional M&E advisor, for example.

![image](https://user-images.githubusercontent.com/33670923/49545453-89772280-f892-11e8-86b8-1c57847d4872.png)

**Comment by jennyhavah** [comment]
The graphic looks good. Here's what I see for a program in Afghanistan where all indicators are LoP only and the program end date has not yet occurred.

![image](https://user-images.githubusercontent.com/33670923/49600675-9ea68c80-f938-11e8-8cb5-5b9f4cec2b15.png)

**Comment by jennyhavah** [comment]
Just realized we have another scenario I hadn't considered that may require a tweak to the language. 

So for the above program, I took one of the indicators that had no targets and set up quarterly targets. However, I did not add any results to that indicator. So the situation we're in now is that the first target period has ended, but the graph says, "Unavailable until the first target period ends."

So @axoplasm, I'd like to request another tweak to the text: 

Unavailable until the first target period ends with results reported
